core/src/banking_stage/unprocessed_transaction_storage.rs

```diff
@@ -364,25 +343,6 @@ impl UnprocessedTransactionStorage {
         }
     }
 
-    pub fn filter_forwardable_packets_and_add_batches(
```

Comment:
Before forwarder used this to filter outdated transactions, now we skip this. Is this filtering done somewhere upper in the stack now?

Reply:
It's not, we can add addtional filtering in `ForwardingStage`.

Was trying to keep it as simple as possible for v0 because what filtering we actually care about doing is somewhat questionable.

We only forward tpu votes and **partnered** connection's non-votes.
votes I think ideally we'd check age and possibly that they are valid staked votes.
non-votes that we forward are coming from our partnered connections, so these should be somewhat trusted in terms of content...if they are spamming old transactions then they are wasting their money paying me to forward.